### PR TITLE
Add GraphiQL to project and fix some typos

### PR DESCRIPTION
**Pull request by rbalmf**
Added this nice tool to help you test your server: https://github.com/graphql/graphiql

Also changed the RPC-sounding query names `getChannel` to be more like entity names `channel` etc.

Run `yarn` again to install new dependencies, then `yarn start` and go to http://localhost:3001/playground in browser.

Try a query like:

```
{
  channels {
    id
    name
    category
  }
  sportsChannels {
    id
    name
    category
  }
  newsChannels {
    id
    name
    category
  }
}
```

image

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,14 +1,25 @@
...
 const { resolvers } = require("./Schema/Resolvers");
-const express = require("express");
 
-const app = express();
+const PORT = 3001;
+
+const startServer = async () => {
+  const isDev = process.env.NODE_ENV !== "production";
+  const server = new ApolloServer({ typeDefs, resolvers });
+  await server.start();
```
Comment (rbalmf) [inline index.js:14]: It is important to `await server.start()` according to Apollo docs

File: Schema/Resolvers.js [hunk 2]
```diff
@@ -3,24 +3,24 @@ const { channels, basket } = require("../data");
 const resolvers = {
   Query: {
     // Channel
-    getAllChannel() {
+    channels() {
       return channels;
     },
-    getSportsChannel() {
+    sportsChannels() {
```
Comment (rbalmf) [inline Schema/Resolvers.js:9]: This should probably be a parameter on `channels`, but it's okay for now

File: Schema/Resolvers.js [hunk 3]
```diff
@@ -3,24 +3,24 @@ const { channels, basket } = require("../data");
...
       const sportsChannel = channels.filter(
         (channel) => channel.category === "Sports"
       );
       return sportsChannel;
     },
-    getNewsChannel() {
+    newsChannels() {
```
Comment (rbalmf) [inline Schema/Resolvers.js:15]: Same as above, just a filter param on `channels`

File: Schema/Resolvers.js (hunk 2 above)
Comment (StebsAlex) [reply Schema/Resolvers.js:9]: you mean filter by channel and add variables like 

```
getChannel($catergory: String!) {
     getChannel($catergory: category) {
           id
           name
           category
     }
} 
```

Comment (rbalmf) [reply Schema/Resolvers.js:9]: Yeah that's the idea, but with the name `channels`, not `getChannel`.  Note plural and don't think of this as a function call, it's a query for a field so the naming relates to the field type not an imperative/RPC type action (i.e. not verb-y)

**Pull request merged**